frontend/bhili-language-explorer/bhili-language-explorer.css

```
@@ -0,0 +1,1245 @@
+    --bhili-primary-hover: #d35400;
+    --bhili-accent: #f39c12;
+    --bhili-gold: #f1c40f;
+    --bhili-forest: #27ae60;
+    --bhili-teal: #16a085;
+    --bhili-deep-bg: #0f141d;
+    --bhili-card-bg: rgba(22, 30, 46, 0.85);
+    --bhili-card-border: rgba(230, 126, 34, 0.25);
+    --bhili-card-hover-border: rgba(243, 156, 18, 0.6);
+    --bhili-text-main: #f8fafc;
+    --bhili-text-muted: #94a3b8;
+    --bhili-font-main: 'Outfit', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)